dokka-integration-tests/gradle/projects/it-android-0/gradle.properties

```diff
@@ -2,6 +2,6 @@
 # Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
 #
 
-dokka_it_kotlin_version=1.9.22
+dokka_it_kotlin_version=2.0.0-Beta4
```

Comment:
Good question, I have no idea :) We can try to remove it in a separate PR, see if the tests fail

Reply:
I believe the version is set as a default in IT projects, so if someone wants to open them in IJ and edit them, it's easier.

Reply:
But the version is used mostly in `template.settings.gradle.kts` (with small exceptions), so to open the project in IDEA, you will still need to copy it inside project folder, or copy it's content
So, I would propose to remove this (in some future PR), if possible, to reduce amount of places where we need to update Kotlin version (or any other version), so that at some point we will not have some situation, that this version is not updated or not overridden in integration tests